### ISSUE TITLE
 make a CI for checking if user generated the localization resource file

### ISSUE THREAD

**Issue by andy840119** [karaoke-dev/karaoke#1269]
Inspired from #1268

Make the hint like this:
```
run the comment to generate the localization file:
dotnet tool restore
dotnet localisation to-resx ./osu.Game.Rulesets.Karaoke/osu.Game.Rulesets.Karaoke.csproj
```

**Comment by andy840119** [comment]
Also, not really should be better to change the `.resx` file location?
Because not be a good idea to mix the code and resource file.
![image](https://user-images.githubusercontent.com/9100368/162973809-8f2e13e8-260e-41c3-a338-76313e437854.png)

**Comment by andy840119** [comment]
Implemented in the #1369 and duplicated with #1368.
